openwisp-openwrt-monitoring/Makefile

```diff
@@ -0,0 +1,74 @@
+# openwisp.org
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwisp-monitoring
+PKG_VERSION:=$(shell cat ../VERSION)
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)/$(PKG_SOURCE_SUBDIR)
+PKG_LICENSE:=GPL-3.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openwisp-monitoring
+	TITLE:=OpenWISP Monitoring
+	CATEGORY:=Network
+	SECTION:=base
+	SUBMENU:=openwisp
+	PKGARCH:=all
+	MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
+	URL:=http://openwisp.org
+endef
+
+define Package/netjson-monitoring
+	TITLE:=NetJson Monitoring
+	CATEGORY:=Network
+	SECTION:=base
+	SUBMENU:=openwisp
+	DEPENDS:=+libubus-lua +lua-cjson +rpcd-mod-iwinfo
+	PKGARCH:=all
+	MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
+	URL:=http://openwisp.org
```

Comment:
I am trying to split it in two packages but there are some problems in makefile due to which only one package is shown in menuconfig. Will fix it soon

Reply:
Have a look here for a package that is split into multiple packages via one makefile. 
 I believe you may also need to create a [Config.in](https://github.com/openwrt/packages/blob/master/net/modemmanager/Config.in) file. 
 https://github.com/openwrt/packages/blob/master/net/modemmanager/Makefile

Reply:
Hi @nickberry17 sorry I forget to resolve this. This issue was arising due to some dependencies and is solved already.
Also, Config.in is used here to install extra dependencies (if I am not wrong), i.e. `libmbim` and `libqmi`

Reply:
OK, let's find out what is going on with that.  Is it an error related to compiling `rpcd-mod-iwinfo` or an error installing that package into the OpenWrt image? Also, earlier I pointed you to ModemManager when I should have shown you [libqmi](https://github.com/openwrt/packages/blob/master/libs/libqmi/Makefile), sorry!  Libqmi split the utils from the library.
>Also, Config.in is used here to install extra dependencies (if I am not wrong), i.e. libmbim and libqmi

Yes, keep that in mind, you may need it later when you want to break the package up or add nice extra options for the user.

Reply:
I have to select `rpcd-mod-iwinfo` first before selecting for openwisp packages. That solves the issue